### PR TITLE
[MIG] migrate business_requirement_deliverable_project_task_categ from 10.0 to 12.0

### PR DESCRIPTION
**Pull request by ruter-lyu**

### CODE REVIEW

File: business_requirement_deliverable_project_task_categ/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# © 2016-2019 Elico Corp (https://www.elico-corp.com).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Business Requirement Deliverable Project Task Categ",
+    'category': 'Business Requirements Management',
+    'summary': 'Adds Task Categories to your Business Requirement Resources',
+    "version": "12.0.1.0.1",
```
Comment (elicoidal) [inline business_requirement_deliverable_project_task_categ/__manifest__.py:7]:
```suggestion
    "version": "12.0.1.0.0",
```

**Review by elicoidal — COMMENTED** [review]
Please prepare the README.
Almost there

**Comment by elicoidal** [comment]
@JoJoJoJoJoJoJo @joycehuan @seb-elico please have a look

**Review by JoJoJoJoJoJoJo — APPROVED** [review]
Technically LGTM, follow review of @elicoidal

File: business_requirement_deliverable_project_task_categ/README.rst [hunk 2]
```diff
@@ -0,0 +1,81 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
```
Comment (nikul-serpentcs) [inline business_requirement_deliverable_project_task_categ/README.rst:3]: @ruter-lyu  LGPL-3

**Comment by pedrobaeza** [comment]
Closing as old and not green

**Pull request closed without merging**